.cirrus.yml

```diff
@@ -58,6 +58,10 @@ functional_task:
       env:
         DISTRO: rawhide
         TEST_EXTRA_ARGS: "--ovs-track"
+    - name: Centos 8 Stream
+      env:
+        DISTRO: c8s
+        TEST_EXTRA_ARGS: "--ignore=test_ovs.py --ignore=test_nft.py"
```

Comment:
I prepared the same series before the year's end PTOs, but I avoided pushing it because we needed to skip these tests.
TBF, it's ok, but at this point, I'm in favor of prioritizing a bit #72 (at least have some consensus about the approach). Let's sync offline on this, though.

Reply:
marked as resolved as this was just a reminder for the outstanding issue with modules